README.md

```diff
@@ -13,28 +13,28 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 
 ## Contributing to this collection
 
-The content of this collection is made by [people](CONTRIBUTORS) like you, a community of individuals collaborating on making the world better through developing automation software.
+The content of this collection is made by [people](https://github.com/ansible-collections/community.postgresql/blob/main/CONTRIBUTORS) like you, a community of individuals collaborating on making the world better through developing automation software.
 
 All types of contributions are very welcome.
 
-You don't know how to start? Refer to our [contribution guide](CONTRIBUTING.md)!
+You don't know how to start? Refer to our [contribution guide](https://github.com/ansible-collections/community.postgresql/blob/main/CONTRIBUTING.md)!
 
 We use the following guidelines:
 
-* [CONTRIBUTING.md](CONTRIBUTING.md)
+* [CONTRIBUTING.md](https://github.com/ansible-collections/community.postgresql/blob/main/CONTRIBUTING.md)
 * [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html)
 * [Ansible Development Guide](https://docs.ansible.com/ansible/devel/dev_guide/index.html)
 * [Ansible Collection Development Guide](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections)
 
-The current maintainers are listed in the [MAINTAINERS](MAINTAINERS) file. Don't hesitate to reach them out mentioning in the proposals. To learn how to maintain / become a maintainer of this collection, refer to the [Maintainer guidelines](https://github.com/ansible/community-docs/blob/main/maintaining.rst).
+The current maintainers (contributors with `write` or higher access) are listed in the [MAINTAINERS](https://github.com/ansible-collections/community.postgresql/blob/main/MAINTAINERS) file. Don't hesitate to reach them out mentioning in the proposals. To learn how to maintain / become a maintainer of this collection, refer to the [Maintainer guidelines](https://github.com/ansible/community-docs/blob/main/maintaining.rst).
 
 ## Communication
 
-We announce releases and important changes through the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546). Be sure you are subscribed.
+We announce important development changes and releases through the [Ansible Bullhorn newsletter](http://eepurl.com/gZmiEP). If you are a collection developer, be sure you are subscribed.
```

Comment:
I'm not sure if it is a good idea to shorten this link here. It feels a little bit unsecure and hide the source.
Is there a need for shorting the URL?

Reply:
@tcraxs thanks for the note. I think this link was originally provided by the subscription service. I'm not sure if we replace it, it will work later. To make it feeling more secure I changed `http` to `https`:)

Reply:
ah okay, I just found the link in the Issue itself, so it was created by @cybette. But to be honest, I do not like shortening services, because they hide the original source and could be a security Issue. Also we (this ansible collection) do not manage this shortening URL.
We could also change to the original URL: https://ansible.us19.list-manage.com/subscribe?u=56d874e027110e35dea0e03c1&id=d6635f5420 -> also not the coolest option ;)
You can decide, I'm fine with both options.

Reply:
Found a better solution:
Link to this documentation of the Bullhorn: https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn
All necessary infos are listed there :)

Reply:
Ah never mind, the shortening URL is spread over all Ansible Documentations ... so just use it :/

Reply:
Mailchimp manages this shortening url, and if it goes down, we will have to use another delivery option for the newsletter anyway :) But I do prefer linking to the wiki with [Bullhorn newsletter](https://github.com/ansible/community/wiki/News#the-bullhorn) and the shortened link with [subscribed](https://eepurl.com/gZmiEP).

Reply:
I like the idea about putting `https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn` instead. Done, thanks!

Reply:
PTAL

Reply:
@cybette i didn't see your message. If we put the link to the doc, will it be fine?

Reply:
@Andersson007 yes, that's fine too! :+1: